packages/legend-studio/src/stores/LegendStudioPlugin.ts

```diff
@@ -52,6 +56,13 @@ export type ClassPreviewRenderer = (
   _class: Class,
 ) => React.ReactNode | undefined;
 
+export type ModelLoaderExtensionConfiguration = {
+  modelGenerationConfig: ModelGenerationConfiguration;
+  allowHardReplace?: boolean;
+  load: (editorStore: EditorStore) => Promise<void>;
```

Comment:
Let's document what this flag does

Reply:
Generally, I think we should make this interface a little more generic, it basically should control the feature set of `model loader`, for example, it can be used to control whether we enable/support the `load truck` icon.